src/mutable_data.rs

```diff
@@ -864,7 +920,7 @@ impl SeqEntryActions {
         self
     }
 
-    /// Update an existing key-value pair
+    /// Updates an existing key-value pair.
```

Comment:
It's also worth explaining what the `version` parameter means and what it should be equal to.